resources/education.md

```diff
@@ -0,0 +1,46 @@
+---
+parent: Resources
+title: Education
+description: Machine translation learning resources
+---
+
+
+## Machine Translation massive open online course
+
+The **Machine Translation** course is a massive open online course from Coursera.
+This course is taught by Professor Doctor Alexander Waibel and Assistant Professor Doctor Jan Niehus from Karlsruhe Institute of Technology.
```

Comment:
> ... by Alexander Waibel, the co-founder of [KITES](/industry/companies.md#kites) and Jan Niehus.

Reply:
I'd rather not list their titles and affiliations, it's not evergreen.

Also just too German.  We don't say "Professor Doctor Alon Lavie" or "Professor Doctor Philipp Koehn" everywhere they are mentioned, even though it's true.